OSDOCS-8646: Rewriting introduction to networking (ContentX) (#87228)
openshift/openshift-docs: modules/nw-understanding-networking-what-is-a-cluster.adoc

// Module included in the following assemblies:
//
// * networking/understanding-networking.adoc

:_mod-docs-content-type: CONCEPT
[id="nw-understanding-networking-what-is-a-cluster_{context}"]

> = What is a cluster?

Reviewer: ```suggestion
= Definition of a cluster
```

Author: done

> A cluster is a collection of nodes that work together to run containerized applications. These nodes include master nodes and worker nodes. Master nodes manage the cluster and maintain cluster-wide operations. Worker nodes run the containers that house the application workloads. Together, these nodes form a cohesive environment where resources can be efficiently managed, scaled, and distributed according to the needs of the applications running within the cluster.

Reviewer: ```suggestion
A cluster is a collection of nodes that work together to run containerized applications. These nodes include control plane nodes and compute nodes.
```

Reviewer: See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language

Author: Thanks! I think I might've over-modularized these sections.
